.circleci/config.yml

```diff
@@ -345,3 +345,22 @@ workflows:
                 - << pipeline.parameters.dev_git_branch >>
                 - << pipeline.parameters.staging_git_branch >>
                 - << pipeline.parameters.prod_git_branch >>
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 13 * * *"
```

Comment:
> I'm good with only running on weekdays

Did you change your mind about this or did you still want to do `1-5`/weekdays?

Reply:
🤦🏻 https://github.com/adhocteam/Head-Start-TTADP/pull/57